R/btw_this.R

```diff
@@ -0,0 +1,64 @@
+btw_this <- function(x, ...) {
+  UseMethod("btw_this")
+}
+
+#' @export
+btw_this.default <- function(x, ...) {
+  # TODO: Replace with {evaluate}
+  out <- capture.output(print(x))
+  if (length(out) && nzchar(out)) return(out)
+
+  capture.output(print(x), type = "message")
+}
+
+#' @export
+btw_this.data.frame <- function(x, ...) {
+  # Implementation for the data.frame method
+}
+
+#' @export
+btw_this.tbl <- function(x, ...) {
+  # Implementation for the tbl method (same as data.frame)
+}
+
+#' @export
+btw_this.Chat <- function(x, ...) {
+  btw_ignore()
```

Comment:
`btw_ignore()` is a special sentinel to say: ignore this object and don't include it in the description.